### PR TITLE
Initial Commit to Review 

### PR DESCRIPTION
**Pull request by bhskr44**
This is done as a part of the code review task assigned by microverse, where we need to copy the code of the todolist and ask for a peer review.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  populateAllTodo();
+  todoItem.value = '';
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  populateAllTodo();
+});
+
+clearCompleted.addEventListener('click', () => {
+  let todoList = JSON.parse(localStorage.getItem('todo'));
```
Comment (bhskr44) [inline src/index.js:37]: A function can be used in [place of similar lines of code

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import saveTodo from './modules/saveTodo.js';
+import populateTodo from './modules/populateTodo.js';
+
+const mainContainer = document.querySelector('#todo-container');
+const clearCompleted = document.querySelector('.clear');
+
+const populateAllTodo = () => {
+  mainContainer.innerHTML = '';
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+
```
Comment (abdallahmalima) [inline src/index.js:12]: Kindly use the DRY principle on this line of code,👍 you have repeated it at many places, and you can create a function and call it whenever you need it. your code will be straightforward to maintain and scalable🤩

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  let todoList = JSON.parse(localStorage.getItem('todo'));
+
+  todoList = todoList.filter((todo) => todo.isCompleted === false);
+  let i = 1;
+  todoList.forEach((todo) => {
+    todo.index = i;
+    i += 1;
+  });
+  localStorage.setItem('todo', JSON.stringify(todoList));
+  window.location.reload();
```
Comment (abdallahmalima) [inline src/index.js:46]: Kindly don't use window reload,🙏 instead content of the page should change without page reload SPA(single page application)🖐

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @bhskr44 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
The PR title and description text looks good and professional, it describes what you have done in this PR✔️
There are no linter errors👍
The project works like magic👏
Required Changes ♻️

**Pull request closed without merging**